pkg/cloud/services/securitygroup/securitygroups_test.go

```diff
@@ -619,13 +619,13 @@ func TestAdditionalControlPlaneSecurityGroup(t *testing.T) {
 
 	testCases := []struct {
 		name                         string
-		controlPlaneLBSpec           *infrav1.AWSLoadBalancerSpec
+		networkSpec                  infrav1.NetworkSpec
```

Comment:
It seems that you are removing the unit test created to test the control plane load balancer ingress rules. I think that test still makes sense. We need a new test to cover the `additionalControlPlaneIngressRules`, and rename the current test from `TestAdditionalControlPlaneSecurityGroup` to something more appropiate like `TestControlPlaneLoadBalanacerIngressRules`.

Reply:
`TestControlPlaneLoadBalancerIngressRules` exists on line 776

Reply:
Ok, that makes sense.